content/Solutions/Integrations/SMBClustering.md

```diff
@@ -0,0 +1,128 @@
+
+When the SCALE, AD, and TrueCommand environments are ready, log in to TrueCommand to cluster the SCALE systems.
+
+1. Click the <span class="iconify" data-icon="mdi:server-network"></span> **Clusters** icon in the upper left. Click **NEW CLUSTER** to see the cluster creation options.
+
+   ![CreateClusterSystemsNetwork](/images/TrueCommand/2.2/CreateClusterSystemsNetwork.png "Network Options for Clustered Systems")
+
+2. Enter a unique name for the cluster and open the dropdown to select the systems to include in the cluster.
+3. When each SCALE system is listed, open the **Network Address** dropdown for each system and choose the static IP address from the previously configured subnet dedicated for cluster traffic.
+4. Click **NEXT**, verify the settings, then click **CREATE**
+
+It can take an extended amount of time to create the cluster.
+
+After the initial creation step for the cluster, TrueCommand opens another sidebar to configure the new cluster for AD connectivity and SMB sharing:
+
+![ConfigureClusterSMBNetwork](/images/TrueCommand/2.2/ConfigureClusterSMBNetwork.png "Configure Cluster SMB Network")
+
+{{< hint warning >}}
+Skipping this step is not recommended because there are no opportunities to reset the configuration after it is completed.
+To go back and add the AD and SMB connection details requires deleting and remaking the cluster.
+{{< /hint >}}
+
+1. For each SCALE system, choose the IP address related to the primary subnet.
+   This is typically the IP address used to connect the SCALE system to TrueCommand.
+   Click **NEXT**.
+2. Enter the Microsoft Active Directory credentials and click **NEXT**.
+3. Verify the connection details are correct and click **SUBMIT**.
+
+{{< hint info >}}
+Creating a cluster has no visible effect on each SCALE web interface.
+To verify the cluster is created and active, open the SCALE **Shell** and enter `gluster peer status`.
+The command returns the list of SCALE IP addresses and current connection status.
+{{< /hint >}}
+
+## Creating Cluster Volumes
+
+1. In the TrueCommand **Clusters** screen, find the cluster to use and click **CREATE VOLUME**.
+   
+   ![ClustersCreateVolumeDetails](/images/TrueCommand/2.2/ClustersCreateVolumeDetails.png "Add Cluster Volume: Details")
+   
+2. Enter a unique name for the cluster and select a **Type**.
+
+   ### Volume Types
+
+   {{< expand "Current Types (click to expand)" "v" >}}
+   {{< include file="/content/_includes/ClusterTypes.md" type="page" >}}
+   {{< /expand >}}
+
+3. After configuring the **Type**, enter a **Brick Size** based on the available storage from the clustered pools and your storage requirements.
+4. Review the **Pools** for each SCALE system in the cluster and ensure that the desired pool is used for this cluster volume.
+5. Click **NEXT**.
+6. Review the settings for the new volume and click **CREATE** when ready.
+
+New cluster volumes are added to the individual Cluster cards on the TrueCommand **Clusters** screen.
```

Comment:
why is cluster capitalized? if it is a general mention of a cluster don't capitalize it. If you are talking about something they see on the screen called Cluster use bold.